data/lipophilicity/meta.yaml

```diff
@@ -0,0 +1,31 @@
+name: lipophilicity
+description: Experimental results of octanol/water distribution coefficient (logD
+  at pH 7.4).
+targets:
+- id: exp
+  description: experimental results of octanol/water distribution coefficient (logD
+    at pH 7.4)
+  units: logD
+  type: continuous
+  names:
+  - octanol/water distribution coefficient (logD at pH 7.4)
+  - octanol/water distribution coefficient
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+license: CC BY-SA 3.0
+links:
```

Comment:
Shall we leave them like this? 

Originally I thought of adding some more context with "description" etc. 
I'm ok with dropping this (but there should be one link we highlight as the one where the data comes from). 


I also realized that this part is currently not validated.

Reply:
Do you mean the description at the top or for the license or links?

Reply:
what I mean is to replace the `links` with something like 

```
links:
  - name: dataset
    url: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/OVHAW8
    description: Original dataset
```

then we could at least identify what link refers to the original dataset and which are other references.

Reply:
ok, got it, I added a url description but no name field.